views/layout.html

```diff
@@ -29,14 +29,24 @@
 {% block beforeMain %}
   {#  Headers or Navigation  #}
   <div class="absolute w-100 bg-white bg-transparent-ns z-1">
-    <nav class="w-site center w-80-ns">
+    <nav class="w-site center w-80-ns tr">
       <div class="center tc pt3 pl3">
         <a href="{{ data.home._url }}" class="fl">
-          <img src="../images/logo_na_dark.svg" alt="Quodl logo" class="w3-5">
+          {% if data.piece.tags == "" %}
+            <img src="../images/logo_na_dark.svg" alt="Quodl logo" class="w3-5">
+          {% else %}
+            <img src="../../images/logo_na_dark.svg" alt="Quodl logo" class="w3-5">
+          {% endif %}
```

Comment:
@Cleop I feel that there should probably be a nicer solution for this bit.
What is the difference in what should happen in the branches of this `if` statement? Do they both render the same thing?